src/tests/api/DocumentSelectionNegativePath.test.ts

```diff
@@ -19,9 +22,27 @@ describe("E2E Negative Path /documentSelection Endpoint", () => {
 
 	});
 
-	it("E2E Happy Path Journey - Incorrect Session Id", async () => {
+	it("E2E Negative Path Journey - Incorrect Session Id", async () => {
 		const response = await postDocumentSelection(dataPassport, "sessionId");
 		expect(response.status).toBe(401);
 		expect(response.data).toBe("Unauthorized");
 	});
+
+	it("E2E Negative Path Journey - Missing fad_code", async () => {
```

Comment:
I'm being a bit nit picky here but looking at the payloads the would be `empty fad_code` as fad_code is in the payload but doesn't have a value. 

Could we please also have a test for where fad_code is completely excluded? This should trigger the required check at the gateway so I'd expect it to return a 400 bad request

Reply:
The reason I want to make sure it's covered is that this is the only validation that we actually have in our code the other stuff is actually all just errors from yoti that we propogate

Reply:
Additional test added to test against a missing fad_code element